<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsExtentSelector</class>
 <widget class="QWidget" name="GdalToolsExtentSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>134</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the extent by drag on canvas</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>or change the extent coordinates</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="x1CoordEdit"/>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEdit" name="x2CoordEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="y1CoordEdit"/>
      </item>
      <item row="1" column="8">
       <widget class="QLineEdit" name="y2CoordEdit"/>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" rowspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>2</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnEnable">
     <property name="text">
      <string>Re-Enable</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
